internal/pkg/tasks/tasks.go

```diff
@@ -43,15 +43,22 @@ func (t *Task) PerformTasks() {
 		combinedIPPermissions = utils.CombineIpPermission(combinedIPPermissions, ipList)
 	}
 
-	loadBalancerNames := t.getLoadBalancerNames(t.config.Filter)
+	if t.config.Filter.FilterType.String() == config.LoadBalancerStr {
```

Comment:
Perhaps should be `t.config.Filter.FilterType == FilterType.LoadBalancer`

Reply:
yeah I had that like that but had to change since I used camel case for the filter, reverting it